README.md

```diff
@@ -216,3 +218,49 @@ In order to log the transfer of data objects from one tier to the next, set `dat
 },
 ```
 
+## Limitations
+
+There are a few known limitations to the storage tiering plugin which should be noted explicitly for understanding different failure modes which users may experience.
+
+### A data object should only have replicas in one tiering group
+
+Any given data object should only have replicas in a single tiering group. Stated negatively, a data object should NOT have replicas in multiple tiering groups. The tiering group for a data object is tracked by an AVU that looks like this:
```

Comment:
I think the first sentence can be dropped. The second sentence is very clear IMO.

Should we use **must** instead of **should**?
Are these recommendations or rules admins must follow for correct behavior?

Reply:
This is not enforced by the unified storage tiering plugin (see #240), so **must** is not technically accurate. From a behavioral correctness perspective, **must** would be appropriate. :) I don't feel strongly about it, so, I'll change it to whatever seems best.

Reply:
I haven't landed on a concrete answer yet, but overall, I think the wording is fine as is. We can tweak things before a new release if needed.

Let's let it ride.